tests/ext/sandbox-regression/class_resolver_bailout_hook.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Assert bailouts are gracefully handled within class autoloading
+--SKIPIF--
+<?php if (PHP_VERSION_ID >= 70300 && PHP_VERSION_ID < 70400) die('skip: Bailing out in autoloaders is fundamentally broken in PHP 7.3'); ?>
```

Comment:
**question (non-blocking):** What would be the expected behaviour in PHP 7.3, segfault?
Do we document such things somewhere, just in case another customer experiences this problem?

Reply:
Yes, this segfaults. It's plain and simple a bug in PHP 7.3 itself.

For now I'm not aware of such documentation.